envi/archs/ppc/emu.py

```diff
@@ -512,13 +576,13 @@ def _rfi(self, op):
         pass
 
     def _trap(self, op):
-        print('TRAP 0x%08x: %r' % (op.va, op))
+        logger.debug('TRAP 0x%08x: %r', op.va, op)
 
     def _sc(self, op):
-        print('SC 0x%08x: %r' % (op.va, op))
+        logger.debug('SC 0x%08x: %r', op.va, op)
 
     def _ehpriv(self, op):
-        print('EHPRIV 0x%08x: %r' % (op.va, op))
+        logger.debug('EHPRIV 0x%08x: %r', op.va, op)
```

Comment:
because they are system calls, i'd make this `warning` or at least `info`

Reply:
I will change to info